src/hooks/useCard.tsx

```
@@ -0,0 +1,163 @@
+          state,
+          confirmText: '내 모임 보러가기',
+          cancelText: '확인',
+        });
+      case 'registerComplete':
+        setModalValue({
+          state,
+          confirmText: '내 모임 보러가기',
+          cancelText: '확인',
+        });
+    }
+    setIsModalOpen(true);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)